server/.env_sample

```diff
@@ -1,4 +1,4 @@
 THIS IS JUST A FILE TO HELP YOU KNOW WHAT IS NEEDED IN THE .env FILE.
```

Comment:
Remove this file, it's not part of your PR

Reply:
Mam Can You Please Review One More Time Please